Library/Homebrew/development_tools.rb

```diff
@@ -74,6 +74,14 @@ def clang_build_version
         end
     end
 
+    def llvm_clang_build_version
+      @llvm_clang_build_version ||=
+        if Tab.for_name "llvm"
+          path = Formulary.factory("llvm").opt_prefix.join("bin", "clang")
+          `#{path} --version`.split(" ")[2]
```

Comment:
I just changed it to use a regex similar to the one `clang_version` uses. Would that work?